### PR TITLE
Update limits-specifications-teams.md

### PR DESCRIPTION
**Pull request by lindspea**
https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/5263
Added important note.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/limits-specifications-teams.md [hunk 1]
```diff
@@ -104,6 +104,16 @@ Channel names also can't start with an underscore (_) or period (.), or end with
 
 ## Meetings and calls
 
+> [!IMPORTANT]
+>
```
Comment (JohanFreelancer9) [inline Teams/limits-specifications-teams.md:108]:
```suggestion
> In April, [we announced](https://techcommunity.microsoft.com/t5/microsoft-teams-blog/what-s-new-in-microsoft-teams-april-2020/ba-p/1347929) that temporary default limit increases would be enabled for Microsoft 365 live events to help better support customers, through July 1st. To continue meeting the needs of customers, we will extend the temporary default limits until October 1, 2020. Live events hosted in Teams, Stream, and Yammer will continue to temporarily support events for up to 20,000 attendees, 16 hours in length, and 50 events happening simultaneously. Additionally, customers can host a live event in Stream for up to 100,000 attendees when planned though the [Microsoft live events assistance program](https://resources.techcommunity.microsoft.com/live-events/assistance/).
```

File: Teams/limits-specifications-teams.md [hunk 2]
```diff
@@ -104,6 +104,16 @@ Channel names also can't start with an underscore (_) or period (.), or end with
...
+In April, [we announced](https://techcommunity.microsoft.com/t5/microsoft-teams-blog/what-s-new-in-microsoft-teams-april-2020/ba-p/1347929) that temporary default limit increases would be enabled for Microsoft 365 live events to help better support customers, through July 1st.
```
Comment (JohanFreelancer9) [inline Teams/limits-specifications-teams.md:109]:
```suggestion
> In April, [we announced](https://techcommunity.microsoft.com/t5/microsoft-teams-blog/what-s-new-in-microsoft-teams-april-2020/ba-p/1347929) that temporary default limit increases would be enabled for Microsoft 365 live events to help better support customers, through July 1st.
```

File: Teams/limits-specifications-teams.md [hunk 3]
```diff
@@ -104,6 +104,16 @@ Channel names also can't start with an underscore (_) or period (.), or end with
...
+
+To continue meeting the needs of customers, we will extend the temporary default limits until October 1, 2020.
```
Comment (JohanFreelancer9) [inline Teams/limits-specifications-teams.md:111]:
```suggestion
> To continue meeting the needs of customers, we will extend the temporary default limits until October 1, 2020.
```

File: Teams/limits-specifications-teams.md [hunk 4]
```diff
@@ -104,6 +104,16 @@ Channel names also can't start with an underscore (_) or period (.), or end with
...
+
+Live events hosted in Teams, Stream, and Yammer will continue to temporarily support events for up to 20,000 attendees, 16 hours in length, and 50 events happening simultaneously.
```
Comment (JohanFreelancer9) [inline Teams/limits-specifications-teams.md:113]:
```suggestion
> Live events hosted in Teams, Stream, and Yammer will continue to temporarily support events for up to 20,000 attendees, 16 hours in length, and 50 events happening simultaneously.
```

File: Teams/limits-specifications-teams.md [hunk 5]
```diff
@@ -104,6 +104,16 @@ Channel names also can't start with an underscore (_) or period (.), or end with
...
+
+Additionally, customers can host a live event in Stream for up to 100,000 attendees when planned though the [Microsoft live events assistance program](https://resources.techcommunity.microsoft.com/live-events/assistance/).
```
Comment (JohanFreelancer9) [inline Teams/limits-specifications-teams.md:115]:
```suggestion
> Additionally, customers can host a live event in Stream for up to 100,000 attendees when planned though the [Microsoft live events assistance program](https://resources.techcommunity.microsoft.com/live-events/assistance/).
```

File: Teams/limits-specifications-teams.md [hunk 6]
```diff
@@ -104,6 +104,16 @@ Channel names also can't start with an underscore (_) or period (.), or end with
 
 ## Meetings and calls
 
+> [!IMPORTANT]
+>
+In April, [we announced](https://techcommunity.microsoft.com/t5/microsoft-teams-blog/what-s-new-in-microsoft-teams-april-2020/ba-p/1347929) that temporary default limit increases would be enabled for Microsoft 365 live events to help better support customers, through July 1st.
+
```
Comment (JohanFreelancer9) [inline Teams/limits-specifications-teams.md:110]:
```suggestion
>
```

File: Teams/limits-specifications-teams.md [hunk 7]
```diff
@@ -104,6 +104,16 @@ Channel names also can't start with an underscore (_) or period (.), or end with
...
+To continue meeting the needs of customers, we will extend the temporary default limits until October 1, 2020.
+
```
Comment (JohanFreelancer9) [inline Teams/limits-specifications-teams.md:112]:
```suggestion
>
```

File: Teams/limits-specifications-teams.md [hunk 8]
```diff
@@ -104,6 +104,16 @@ Channel names also can't start with an underscore (_) or period (.), or end with
...
+Live events hosted in Teams, Stream, and Yammer will continue to temporarily support events for up to 20,000 attendees, 16 hours in length, and 50 events happening simultaneously.
+
```
Comment (JohanFreelancer9) [inline Teams/limits-specifications-teams.md:114]:
```suggestion
> 
```

File: Teams/limits-specifications-teams.md (hunk 1 above)
Comment (JohanFreelancer9) [inline Teams/limits-specifications-teams.md:108]:
```suggestion
> In April, [we announced](https://techcommunity.microsoft.com/t5/microsoft-teams-blog/what-s-new-in-microsoft-teams-april-2020/ba-p/1347929) that temporary default limit increases would be enabled for Microsoft 365 live events to help better support customers, through July 1st. To continue meeting the needs of customers, we will extend the temporary default limits until October 1, 2020. Live events hosted in Teams, Stream, and Yammer will continue to temporarily support events for up to 20,000 attendees, 16 hours in length, and 50 events happening simultaneously. Additionally, customers can host a live event in Stream for up to 100,000 attendees when planned though the [Microsoft live events assistance program](https://resources.techcommunity.microsoft.com/live-events/assistance/).
```

File: Teams/limits-specifications-teams.md (hunk 2 above)
Comment (JohanFreelancer9) [inline Teams/limits-specifications-teams.md:109]:
```suggestion
```

File: Teams/limits-specifications-teams.md (hunk 6 above)
Comment (JohanFreelancer9) [inline Teams/limits-specifications-teams.md:110]:
```suggestion
```

**Comment by scanum** [comment]
@LolaJacobsen PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**